### ISSUE TITLE
Impossible to make a dye with only ItemBuilder.

### ISSUE THREAD

**Issue by ItsArt3x54** [Raft08/CraftyBuilder#1]
normaly to make a dye you do:

```java
 ItemStack item1 = new ItemStack(Material.INK_SACK);
 Dye dye = new Dye();
 dye.setColor(DyeColor.LIME);
 item1.setData(dye);
```

but its **not possible** if you make one class with all itemstacks its not possible to make a dye.

**Comment by BelgianDev** [comment]
Will be fixed in the next version.

**Comment by ItsArt3x54** [comment]
Do it with potion too please

**Comment by BelgianDev** [comment]
Fixed #3
